cmd/nri-kubernetes/main.go

```diff
@@ -192,7 +193,12 @@ func runScrapers(c *config.Config, ksmScraper *ksm.Scraper, kubeletScraper *kube
 	if c.Kubelet.Enabled {
 		err := kubeletScraper.Run(i)
 		if err != nil {
-			return fmt.Errorf("retrieving kubelet data: %w", err)
+			if kubeletScraper.IsMaxRerunReached(kubeletReruns) {
+				return fmt.Errorf("retrieving kubelet data: %w", err)
+			} else {
```

Comment:
looks like we can omit the else statement since the if above has a return? can help readability a bit. overall looks good!

Reply:
thanks, remove the else